docs/UserGuide.md

```diff
@@ -148,8 +119,7 @@ Acceptable values for the keywords:
 * `ALL` (shows all student detail fields)
 
 Examples:
-* `list` displays the `SCHEDULE` list with all the lessons.
-* `list schedule` displays all the lessons with their `NAME`.
+* `list` and `list schedule` displays the `SCHEDULE` list with all the lessons with their `NAME`.
```

Comment:
+ "in time order"

Reply:
Made the necessary changes in the recent commit. Thank you!